cafebabel/static/js/script.js

```diff
@@ -113,3 +113,38 @@ Array.from(document.querySelectorAll('h1.edit input')).forEach(inputh1 => {
     () => inputh1.value || h1.classList.remove('active')
   )
 })
+
+/* Auto-expand textarea when typing http://jsfiddle.net/hmelenok/WM6Gq/ */
+var textAreaAutoResize
+if (window.attachEvent) {
+  textAreaAutoResize = function(element, event, handler) {
+    element.attachEvent('on' + event, handler)
+  }
+} else {
+  textAreaAutoResize = function(element, event, handler) {
```

Comment:
write as done elsewhere `function textAreaAutoresize(…` and you can get rid of `var textAreaAutoResize`